Make Randi WT in complete animal the default

lparsons opened the pull request:
Make the default dataset the Randi et al 2022 WT dataset in the complete animal and not the head ganglia.

Resolves #113

@bdsinger You're welcome to review if you like, but not necessary.

lparsons committed:
Make Randi WT in complete animal the default

Make the default dataset the Randi et al 2022 WT dataset in the complete
animal and not the head ganglia.

Resolves #113

hepcat72 commented on src/client/js/controller/index.js:
```diff
@@ -459,11 +459,11 @@ class Controller extends EventEmitter {
     let datasets = intersection(allDatasets, state['datasets'] || []);
     if (datasets.length === 0) {
-      datasets = (database == 'head') ? ['randi_funconn_wildty'] : allDatasets;
+      datasets = (database == 'complete') ? ['randi_funconn_wildcp'] : allDatasets;
```
Would this make more sense?
```
      datasets = (database == 'head') ? ['randi_funconn_wildty'] : ((database == 'complete') ? ['randi_funconn_wildcp'] : allDatasets);
```

lparsons replied:
That's a good point Rob. I'm not sure what we ultimately want, but I like the idea of making the Randi et al datasets the single default for each group.

However, I'm not actually sure how to do that, since this line only executes when there are no previously defined datasets. Once you hit the page for the first time, there is a `datasets.length` is no longer zero. You end up getting all datasets in the `head` group if you switch. I'll see if I can find a simple way to achieve that, but if not, I'll merge this and create a new issue.

hepcat72 replied:
Regarding our lunchtime discussion, doesn't this line mean that the length of the datasets array depends on the state of the selections:
```
let datasets = intersection(allDatasets, state['datasets'] || []);
```

lparsons replied:
Well, I tried for a bit to get this to work, but didn't have any luck. With this PR, the behavior is to start with Complete animal, Randi et al 2022 dataset only. For the other groups, all datasets are select. After the first visit, it uses whatever datasets ere last selected.

I'm happy to change this if we figure it out, but I'm suggesting we leave it as is for now.

hepcat72 reviewed (approved):
I haven't tested my suggestion, so I didn't make it a "suggested" code block.  But in the startup-case, this will behave as expected.  My only concern is the case when you switch back to head and perhaps unexpectedly getting all datasets.

lparsons committed:
Set default datasets for each database

Allow setting of default datasets for each database category (head,
complete, tail, unc-31).

Resolves #113
